lib/node_modules/@stdlib/repl/lib/auto_close_pairs.js

```diff
@@ -65,75 +65,77 @@ function AutoCloser( rli ) {
 *
 * @name beforeKeypress
 * @memberof AutoCloser.prototype
-* @param {string} data - input data
+* @param {string} input - input data
 * @param {Object} key - key object
 * @returns {boolean} boolean indicating whether auto-delete was successful
 */
-setNonEnumerableReadOnly( AutoCloser.prototype, 'beforeKeypress', function beforeKeypress() {
+setNonEnumerableReadOnly( AutoCloser.prototype, 'beforeKeypress', function beforeKeypress( input, key ) {
 	var cursor;
 	var line;
 	var data;
 	var node;
 	var ast;
 	var ch;
 
-	if ( this._ignoreBackspace ) {
-		// Reset the backspace flag to re-enable auto-deletion behavior:
-		this._ignoreBackspace = false;
-		return false;
-	}
-	cursor = this._rli.cursor;
-	line = this._rli.line;
-
-	debug( 'Expression: %s', line );
-	debug( 'Cursor position: %d', cursor );
-
-	debug( 'Performing auto-delete...' );
-
-	data = line[ cursor-1 ];
-	debug( 'Character to delete: %s', data );
-
-	debug( 'Checking if an opening symbol...' );
-	ch = OPEN_SYMBOLS[ data ];
-	if ( isString( ch ) ) {
-		debug( 'Detected an opening symbol.' );
-
-		debug( 'Checking if immediately followed by a corresponding closing symbol...' );
-		if ( ch !== line[ cursor ] ) {
-			debug( 'Opening symbol is not followed by a corresponding closing symbol. Skipping...' );
-			debug( 'Finished performing auto-delete.' );
+	if ( key && key.name === 'backspace' ) {
```

Comment:
Rather than perform extra indentation like this; prefer early return. E.g.,

```
if ( !key || key.name !== 'backspace' ) {
    return false;
}
```

In general, we try and keep significant code blocks as left-aligned as possible, and, here, it better signifies intent. Namely, if the key is not a backspace, we don't need to process.

Reply:
I'll go ahead and make this change, so as to avoid merge conflicts.